login-workflow/src/screens/refactoredSubscreens/ExistingAccountRegisterSuccess.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useLanguageLocale } from '@brightlayer-ui/react-auth-shared';
+import { FinishState } from '../../components';
+import Person from '@mui/icons-material/Person';
+import { BrandedCardContainer, BrandedCardContainerProps } from '../../components/BrandedCardContainer';
+import Divider from '@mui/material/Divider';
+import CardHeader from '@mui/material/CardHeader';
+import Typography from '@mui/material/Typography';
+import { DialogActionsStyles, DialogButtonStyles, DialogContentStyles, DialogTitleStyles } from '../../styles';
+import { useTheme } from '@mui/material/styles';
+import CardContent from '@mui/material/CardContent';
+import CardActions from '@mui/material/CardActions';
+import Button from '@mui/material/Button';
+import { useRoutes } from '../../contexts/RoutingContext';
+import { useNavigate } from 'react-router-dom';
+
+/**
+ * Component that renders a screen displaying success for creating an account for
+ * a user that already exists.
+ *
+ * @category Component
+ */
+
+type ExistingAccountRegisterSuccessProps = BrandedCardContainerProps & {
```

Comment:
Why would you need `BrandedCardContainerProps`, if you are trying to create a subscreen?

Reply:
I am trying to create a Screen, but still not using any of the prop from `BrandedCardContainerProps`. As per the discussion I had with Joe, added this in the code.

Reply:
I do not understand. You are not using any prop from `BrandedCardContainerProps` and are not doing any prop forwarding, why is it here?

Reply:
I have removed `BrandedCardContainerProps` in the latest commit.